stock_carrier_info/models/account_invoice.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
The content of this file should be moved to account_invoice_validate_send_email.  Pay attention to the comment "The module does not contain any business logic per se."

Reply:
Thank you, I will be careful.
Moved logics to account_invoice_validate_send_email and it worked as expect.